CCT_cold/config.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "Columbia Card Task Cold Version",
+        "run": [
+                 "experiment.js",
+                 "style.css"
+               ],
+        "tag": "columbia_card_task_cold",
```

Comment:
to be more specific, the tag variable must correspond to the folder name, which must be lowercase, no special characters. etc